src/routes/github/create-branch/github-create-branch-options-get.ts

```diff
@@ -0,0 +1,61 @@
+import { NextFunction, Request, Response } from "express";
+import { Errors } from "config/errors";
+import { Subscription } from "~/src/models/subscription";
+import { GitHubServerApp } from "~/src/models/github-server-app";
+
+export const GithubCreateBranchOptionsGet = async (req: Request, res: Response, next: NextFunction): Promise<void> => {
+
+	const { jiraHost } = res.locals;
+	const { issue_key: key } = req.query;
```

Comment:
/nit less generic name
```suggestion
	const { issue_key: issueKey } = req.query;
```